src/cf-core/src/cf-media-queries.less

```diff
@@ -29,6 +29,7 @@
     }
 }
 
+// TODO: Discuss whether to split this into min and max queries.
 .respond-to-dpi(@ratio, @rules) {
```

Comment:
Should this file follow the same spacing guidelines for variables in parenthesis that the other files do?